pkg/materialize/generic.go

```diff
@@ -161,3 +161,8 @@ func (b *Builder) alterDrop(name string, options []string, validate bool) error
 
 	return b.exec(query)
 }
+
+func (b *Builder) alterFrom(name string, from IdentifierSchemaStruct) error {
+	query := fmt.Sprintf(`ALTER %s %s SET FROM %s;`, b.entity, name, from.QualifiedName())
```

Comment:
Does `QualifiedName` handle escaping the input?

Reply:
Yep!

```go
func QuoteIdentifier(input string) string {
	return `"` + strings.Replace(input, `"`, `""`, -1) + `"`
}

func QualifiedName(fields ...string) string {
	var o []string
	for _, f := range fields {
		c := QuoteIdentifier(f)
		o = append(o, c)
	}

	q := strings.Join(o[:], ".")
	return q
}
```